src/main/java/seedu/duke/Ui.java

```diff
@@ -40,6 +44,9 @@ public void nextInput(String input, TaskList tasks) {
             case DELETE:
                 myPrint(tasks.delete(Parser.getIndex(input)));
                 break;
+            case FIND:
+                myPrint(tasks.find(Parser.getDescription(input)));
```

Comment:
Maybe it will be better to change the myprint() method name to something that would explain the function better